code/modules/admin/panels/ban_panel.dm

```diff
@@ -308,7 +308,7 @@
 				"}
 				break_counter++
 			output += "</div></div>"
-		var/list/long_job_lists = list("Role Positions" = list(ROLE_XENOMORPH, ROLE_XENO_QUEEN, ROLE_SURVIVOR, ROLE_ERT))
+		var/list/long_job_lists = list("Role Positions" = list(ROLE_XENOMORPH, ROLE_XENO_QUEEN, ROLE_ERT))
```

Comment:
Should xeno still be here?

Reply:
I'll remove it, but it's something I cannot test locally, without a DB.

Reply:
If you want to, i can give you a db to use.

Reply:
That'd be useful